overlord/snapstate/snapstate_test.go

```diff
@@ -1100,7 +1100,7 @@ func (s *snapmgrTestSuite) TestRemoveDiskSpaceForSnapshotError(c *C) {
 	c.Check(diskSpaceErr.ChangeKind, Equals, "remove")
 }
 
-func (s *snapmgrTestSuite) TestRemoveDiskSpaceForSnapshotNotCheckedWhenSnapshotsDisabled(c *C) {
+func (s *snapmgrTestSuite) TestRemoveDiskSpaceCheckDisabled(c *C) {
```

Comment:
what's happening here, did you switch the two tests around? the diff is a bit confusing

Reply:
Yes, that happended when I merged master (with already landed remove bit) back and deconflicted it.

Reply:
That's no longer an issue in this PR.